subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -189,6 +189,8 @@
 #define EVENT_IFS_MAX_US        (EVENT_IFS_US + EVENT_CLOCK_JITTER_US)
 /* Controller will layout extended adv with minimum separation */
 #define EVENT_MAFS_US           300
+/* Controller dependent minimum MAFS used to populate aux_offset */
```

Comment:
```suggestion
/* Controller dependent Minimum AUX Frame Space (MAFS) minimum used to populate aux_offset */
```

Though it sounds weird to have a value with the word "Minimum" in it twice

Reply:
Since MAFS is a minimum, and it shall be 300 US as per the core spec:

> The Minimum AUX Frame Space is designated “T_MAFS” and shall be 300 µs.

What is the difference between `EVENT_MAFS_US` and `EVENT_MAFS_MIN_US`?

Reply:
`EVENT_MAFS_US` as you mention is the Spec defined value that all receivers have to comply.

`EVENT_MAFS_MIN_US` is being used as the minimum used in the calculation/scheduling of Auxiliary PDU by the advertising role. May be renaming to `EVENT_MAFS_ADV_US` would be better?

Reply:
When will `EVENT_MAFS_US` and `EVENT_MAFS_MIN_US` have different values? 

> EVENT_MAFS_US as you mention is the Spec defined value that all receivers have to comply.

Can you share that how you came to that conclusion? I don't read it to only apply to receivers

Reply:
```
The Aux Offset shall be at least the length of the packet plus T_MAFS
```

The Aux Offset is populated by the advertiser and to be complied by the receiver (Scanner/Observer) to receive the auxiliary PDU. 

> When will EVENT_MAFS_US and EVENT_MAFS_MIN_US have different values?

For now, they are same but if users require then this may be converted to Kconfig option so that peer implementations that are not able to support 300 us T_MAFS can have an advertising implementation (ZLL) use >= 300 us in the calculation of aux offset.

Reply:
> The Aux Offset shall be at least the length of the packet plus T_MAFS

Aux Offset != Minimum AUX Frame Space though, so if you want to introduce a new minimum offset, then perhaps call it `EVENT_AO_MIN_US` or `EVENT_AUX_OFFSET_MIN_US` or something like that, possibly with a comment or BUILD_ASSERT that ensures that the new value is always at least `EVENT_MAFS_US` (ideally also with the above quote from the spec).

Reply:
> introduce a new minimum offset

As the length of the packet is not constant at compile time, can not have the minimum offset as the constant and has to be a value corresponding to T_MAFS be used in the calculation of the offset.

Updated comments in code and kept the define same.

Reply:
I would still disagree with the name. `MAFS_MIN` is not a name that really makes sense. 

The `EVENT_MAFS_MIN_US` macro is, as you pointed out yourself, related to `MAFS` but isn't a Frame Spacing value but rather an offset, and should thus not contain the `MAFS` infix IMO.

Reply:
> The EVENT_MAFS_MIN_US macro is, as you pointed out yourself, related to MAFS but isn't a Frame Spacing value but rather an offset, and should thus not contain the MAFS infix IMO.

This is not what I stated, I said "has to be a value corresponding to T_MAFS be used in the calculation of the offset."

I can use `EVENT_ADV_MAFS_US`, similar to `EVENT_B2B_MAFS_US` and `EVENT_SYNC_B2B_MAFS_US`, ok?

Reply:
I don't think the word `MAFS` should be used for a value that isn't the `MAFS`. 

As mentioned the `MAFS` is a specific value defined by the core spec. Any values derived from it, such as the Aux Offset, should IMO not contain the word `MAFS`, as those values are not the minimum aux frame space. 

We may want another set of eyes/opinion here, as I don't want to be blocking anything if I'm the minority :)

Reply:
@wopu-ot Please check if you can get some review here. This PR introduces provision to be able to configure the used MAFS by advertiser, and also fixes a bug where advertiser has been using lower than 300 us in the Nordic's implementation.

This PR blocks #76619 which is the corresponding change in scanner to skip advertisers using lower than 300 us MAFS.